build_docs.py

```diff
@@ -867,10 +860,7 @@ def copy_build_to_webroot(self, http: urllib3.PoolManager) -> None:
                 [
                     "cp",
                     "-a",
-                    *[
-                        str(dist)
-                        for dist in (Path(self.checkout) / "Doc" / "dist").glob("*")
-                    ],
+                    *(self.checkout / "Doc" / "dist").glob("*"),
                     target / "archives",
                 ]
```

Comment:
`run()` calls `str()` on each item of the command list, no need to duplicate here.